File: 2149-rearrange-array-elements-by-sign/2149-rearrange-array-elements-by-sign.py
class Solution:
    def rearrangeArray(self, nums: List[int]) -> List[int]:
        nex=1
        cur=0
        ans = [0]*len(nums)
        for i in nums:
            if "-" in str(i):
                ans[nex] = i
                nex += 2
            else:
                ans[cur] = i
                cur += 2
        return ans

File: 2149-rearrange-array-elements-by-sign/2149-rearrange-array-elements-by-sign.cpp
class Solution {
public:
    vector<int> rearrangeArray(vector<int>& nums) {
        vector<int> ans(nums.size(),0);
        int cur = 0; int nex = 1;int i=0;
        for(auto it:nums){
            if(it<0){
                ans[nex]=it;
                nex+=2;
            }
            else if(it>0){
                ans[cur]=it;
                cur+=2;
            }
        }
        return ans;
    }
};